src/cli.ts

```diff
@@ -80,6 +81,39 @@ export class CliImpl implements Cli {
       });
     });
   }
+
+  async runWatchCommand(cmd: CliCommand, opts: SpawnOptions = {}): Promise<CliExitData> {
+    return new Promise<CliExitData>((resolve) => {
+      this.tknChannel.print(cliCommandToString(cmd));
+      if (opts.windowsHide === undefined) {
+        opts.windowsHide = true;
+      }
+      if (opts.shell === undefined) {
+        opts.shell = true;
+      }
+      const tkn = spawn(cmd.cliCommand, cmd.cliArguments, opts);
+      let stdout = '';
+      let error: string | Error;
+      tkn.stdout.on('data', (data) => {
+        stdout += data;
+        const regexStatus = /\sSucceeded\s|\sFailed\s/;
```

Comment:
This is hardcoded regex, are you sure that it will fit in any case? For any command output?